R/result.R

```diff
@@ -1,36 +1,285 @@
+# Version of R6 with heritable static/class methods and attributes
+#
+# Elements in `static` can be called without instantiation, e.g. `Class$method()`.
+# Functions are evaluated in the environment of `Class`, so you can refer to `self`
+# (which is the class—not the instance—here) to create class methods.
+#' @include util.R
+R6.1Class <- function(..., static = NULL) {
```

Comment:
This is really nitpicky; feel free to disregard. But I've been trying to grok this for a while as "the R6 method of 1Class classes," in the style of e.g. `summary.Date()` or whatever. Part of that is definitely R's fault in overloading `m.c` to mean either "the m method of c classes" or "the m_c variable," exacerbated by the fact that I haven't read R in over a year, but maybe it would be a bit clearer to call this something else? I still like the 6.1 reference, so maybe `R6Point1Class`?

Reply:
Oh yes I like that